app/views/foods/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<div>
+  <h1>Food List</h1>
+  <%= link_to 'Add Food', new_food_path %>
+</div>
+
+<% if @foods.empty? %>
+  <p>Please add foods using the Add Food button</p>
+<% else %>
+<table>
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Measurement Unit</th>
+      <th>Unit Price</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @foods.each do |food| %>
+      <tr>
+        <td><%= link_to "#{food.name}", food_path(food.id) %></td>
+        <td><%= food.measurement_unit %></td>
+        <td><%= food.quantity %></td>
+        <td>$<%= food.price %></td>
+        <td><%= button_to "Delete", food_path(food.id), method: :delete, data: { confirm: 'You are about to delete this food! Do you want to continue?' }, class: 'btn btn-danger delete-button' %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+<% end %>
```

Comment:
- [ ] It would be nice if we keep to the given wireframe
your design :point_down: 
![Screenshot from 2023-12-02 12-32-17](https://github.com/Kaiserabbas/food_recipe/assets/66097371/a2180216-963a-4696-952e-607d4688d99f)
given wireframe :point_down: 
![Screenshot from 2023-12-02 12-36-14](https://github.com/Kaiserabbas/food_recipe/assets/66097371/53d91ae4-511d-4fc1-bdd7-ff0fe49d13ed)